<launch>
    <arg name="interval_radius" default="250"/>
    <group ns="front_ranges">
        <param name="points_from_center" value="$(arg interval_radius)"/>
        <param name="min_range" value="0.3"/>
        <param name="max_range" value="10.0"/>
        <node name="front_ranges_node" pkg="pointcloud_tests" type="front_ranges_node" output="screen"/>
    </group>
</launch>